tests/functional/test_basic.py

```diff
@@ -332,9 +332,20 @@ def MyEXE(*args, **kwargs):
     _, err = capsys.readouterr()
     assert "'import warnings' failed" not in err
 
-@skipif_win
-def test_python_makefile(pyi_builder):
-    pyi_builder.test_script('pyi_python_makefile.py')
```

Comment:
The script file itself should probably be removed as well.

Reply:
Oops good point.